### ISSUE TITLE
Fully test health buttons

### ISSUE THREAD

**Issue by DinkyInkyDog** [DinkyInkyDog/dndtracker#2]
### Test: true/false

damage health: true
current health !below 0: true
heal health: true
current health !above maxhealth: true

fully heal: **false**
_button not present_
instant death: **false**
_button not present_

**notes**

- [x] the current health icon is off centered and needs to be slightly raised up so that it can be more readable. 

![Image](https://github.com/user-attachments/assets/b9a53bbb-d24f-40d8-a562-35a31a4c1843)

- [x] I'd also like to make it so when you hit the button to either add or subtract to the current health, the number field goes back to being blank so you don't have to keep highlighting the number to change it.

**Comment by DinkyInkyDog** [comment]
I took care of the hope that the number field would empty after it was activated. My method worked when I tested it so check that off the list. 

![Image](https://github.com/user-attachments/assets/96ebaa59-ccf4-4671-9471-050efa0d2180)

**Comment by DinkyInkyDog** [comment]
I fixed the current health counter. I changed the size of it and the placement of the number. I'm pretty happy of how it turned out. I had to play around with the css. I also changed the placement of the health controls before I realized that I didn't want those centered on the current health because I still need to add the progress bar.

![Image](https://github.com/user-attachments/assets/58a22e76-274f-4b87-b572-3d049075e4dc)
Here's the css that I ended with that left me with this setup:

![Image](https://github.com/user-attachments/assets/7dd6ea90-4655-4703-935d-4350f209fd89)



now all that's left is to add those extra functions that I didn't have before. I'll make that into a separate issue section though for clarity.
